UnityProject/Assets/Scripts/Objects/Machines/DryingRack.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+using Systems.Electricity;
+using AddressableReferences;
+using Effects;
+using Items;
+using Machines;
+using Objects.Machines;
+
+namespace Objects.Kitchen
+{
+	public class DryingRack : NetworkBehaviour
+	{
+		private RegisterTile registerTile;
+		private ItemStorage storage;
+		public Vector3Int WorldPosition => registerTile.WorldPosition;
+
+		// Checks the first unfilled slot. If the first slot is unfilled then the processor
+		// must be empty. (Always adds starting from the first slot, and emptying always
+		// empties the entire inventory.)
+		public bool IsFilled => (storage.GetNextFreeIndexedSlot().SlotIdentifier.SlotIndex != 0);
+
+		private void Awake()
+		{
+			registerTile = GetComponent<RegisterTile>();
+			storage = GetComponent<ItemStorage>();
+		}
+
+		private void OnEnable()
+		{
+			UpdateManager.Add(CallbackType.UPDATE, UpdateMe);
+		}
+
+		private void OnDisable()
+		{
+			UpdateManager.Remove(CallbackType.UPDATE, UpdateMe);
+		}
+
+		/// <summary>
+		/// Count down processor timer.
+		/// </summary>
+		private void UpdateMe()
+		{
+			CheckDried(Time.deltaTime);
+		}
```

Comment:
Not sure i like this running every frame, maybe every 0.5 seconds would be better?

Reply:
Sounds good to me.